pkg/scd/store/cockroach/availability.go

```diff
@@ -63,7 +63,9 @@ func (u *repo) UpsertUssAvailability(ctx context.Context, s *scdmodels.UssAvaila
 func (u *repo) fetchAvailabilities(ctx context.Context, q dsssql.Queryable, query string, args ...interface{}) ([]*scdmodels.UssAvailabilityStatus, error) {
 	rows, err := q.QueryContext(ctx, query, args...)
 	if err != nil {
-		return nil, stacktrace.Propagate(err, "Error in query: %s", query)
+		if strings.Contains(err.Error(), "does not exist") {
```

Comment:
If `err` is not `nil` and doesn't contain `"does not exist"` (for instance, if there were a SQL syntax error), it seems like execution will continue normally -- is this actually intended behavior?

Reply:
This was a misplaced check. Thank you for finding it.